api/ekirjasto_consortium.py

```diff
@@ -137,6 +137,29 @@ def run_once(self, progress):
             achievements=f"Kirkanta synchronization done!",
         )
 
+    @classmethod
+    @memoize(ttls=86400)  # 24h
+    def get_municipality_code_name_map(cls):
+        """Returns a dict of finnish municipality codes mapped to municipality names
+        from data fetched from Koodistopalvelu. Memoized for 24h.
+        """
+        concept_codes = cls._fetch_koodisto_concept_codes(cls)
+
+        code_name_map: dict[str, str] = {}
+        for item in concept_codes:
+            for attribute in item.attributes:
+                try:
+                    if attribute.attributeName == "LongName":
```

Comment:
Could use findAttribute from KoodistoConceptCode class.

Reply:
I missed that method altogether – changed this to utilize findAtrribute 👍